pkg/chunk/encoding/factory.go

```diff
@@ -15,15 +15,13 @@ type Config struct{}
 
 var (
 	// DefaultEncoding exported for use in unit tests elsewhere
-	DefaultEncoding             = Bigchunk
-	alwaysMarshalFullsizeChunks = true
```

Comment:
Why are you saying it was unused? Being the default `true` doesn't it mean it's always used unless explicitly disabled via CLI flag?

Reply:
Yes, it was written this way to avoid changing things for existing users.
I mean the option to disable was never used.

Reply:
However I got the change wrong.

Reply:
Yes, then we need to remove the flag while always keeping it enabled ;)

Reply:
Try now.